homeassistant/components/nuimo.py

```diff
@@ -0,0 +1,108 @@
+"""
+components.nuimo
+~~~~~~~~~~~~~~~~~~
+Component that connects to a Bluetooth reachable Nuimo device.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/nuimo/
+"""
+from logging import getLogger
+import threading
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.helpers import validate_config
+
+REQUIREMENTS = [
+    'https://github.com/shaftoe/nuimo-raspberrypi-demo/archive/'
+    'pip.zip#nuimo_core==0.0.1',
+    'https://github.com/IanHarvey/bluepy/archive/master.zip#bluepy==1.0.3']
```

Comment:
I've addressed the other issues, you can see the progress here: https://github.com/balloob/home-assistant/commit/ebd15f4cc7a1f6c336cb580268682ff9ae35cd23

I'm currently trying to find out how to implement your sensors suggestion, makes sense.

Unfortunately I'm still not sure how to fix the REQUIREMENTS request, I'm in contact with upstream dev and asking them to take care of this but can't say how long it will take